hive-plugins/src/main/java/co/cask/hydrator/plugin/batch/HiveConfig.java

```diff
@@ -50,6 +52,13 @@ public HiveConfig() {
     dbName = "default";
   }
 
+  @Name(Hive.SCHEMA)
+  @Description("Optional schema to use while reading (in case of Source) or writing (in case of Sink) Hive table. " +
```

Comment:
The description is user facing and should not have this type of language in it. This means there is code duplication, but don't think there's anything we can do about that.

Reply:
Done.